app/src/services/auth-services.ts

```diff
@@ -53,24 +53,22 @@ export default class AuthServices {
    * Throws on error.
    */
   static async fetchUserInfo (): Promise<any> {
-    // PREVENT_STORAGE_SYNC flag is set during signout in sbc common components
-    // when session is state cleared, as a mechanism to prevent returning async
-    // functions to set tokens
-    if (sessionStorage.getItem('PREVENT_STORAGE_SYNC')) return null
-    const url = `${this.authApiUrl}/users/@me`
+    // don't fetch user info if session isn't synced (ie, user is not logged in)
+    // to pre-empt a console error and Sentry log
+    if (sessionStorage.getItem('SESSION_SYNCED') !== 'true') return null
```

Comment:
I'm checking this other session variable, same as in auth-web, ie.

![image](https://github.com/user-attachments/assets/ce7fc67a-11fe-4132-8ced-d177680544f3)